tests/ui/inline-const/break-inside-inline-const-issue-128604.stderr

```diff
@@ -0,0 +1,33 @@
+error[E0268]: `break` outside of a loop or labeled block
+  --> $DIR/break-inside-inline-const-issue-128604.rs:15:9
+   |
+LL |         break;
+   |         ^^^^^ cannot `break` outside of a loop or labeled block
```

Comment:
Was stderr file generated *after* the code changes were made? I'm just surprised that we seem to have the correct behavior here and no changes on the next commit.

Reply:
Ah yes, I generated the tests after the changes. Will make sure to do it properly in the future.